apps/evaluations/tests/test_llm_evaluator.py

```diff
@@ -59,11 +59,11 @@ def test_running_evaluator(get_llm_service, llm_provider, llm_provider_model):
 
     evaluation_run = EvaluationRun.objects.create(team=evaluation_config.team, config=evaluation_config)
 
-    for message in dataset.messages.all():
+    for message in dataset.messages.all().order_by("created_at"):
```

Comment:
I'm not totally sure why this started failing in this PR, which didn't touch any model code or anything. I also wasn't able to reproduce the failures locally... 👻